dev/start

```diff
@@ -67,19 +94,21 @@ while [ $# -gt 0 ]; do
             shift
             ;;
         -n|--dry-run)
-            _DRYRUN=0
+            _DRYRUN="true"
             shift
             ;;
         -h|--help)
             # since zsh is MacOS standard fallback to $0 if not in bash
             echo "Usage: source ${BASH_SOURCE:-$0} [options]"
             echo ""
             echo "Options:"
-            echo "  -p, --python  VERSION  Python version for the env to activate. (default: 3.8)"
-            echo "  -u, --update           Force update packages. (default: update every 24 hours)"
+            echo "  -p, --python  VERSION  Python version for the env to activate. (default: ${_DEFAULT_PYTHON})"
+            echo "      --mach    MACHINE  Python version for the env to activate. (default: ${_DEFAULT_MACH})"
+            echo "      --arch    ARCH     Python version for the env to activate. (default: ${_DEFAULT_ARCH})"
```

Comment:
Just out of curiosity, is there a reason not to allow `-m` and `-a` as well?

Reply:
social engineering, I tend to only expose the short options for those that I want users to use frequently, if there's only a long option I interpret it as "think again" and "read the docs before proceeding"

Reply:
Hmm, I get that, it's a common pattern in end user CLIs, but given that `dev/start` is hardly a tool used by end users, I think it's fair to set them up for ease of use instead of adding a barrier. That said, that's just my thoughts, not really strongly opinionated about it.